skimage/draw/_polygon2mask.py

```diff
@@ -12,7 +12,7 @@ def polygon2mask(image_shape, polygon):
         The shape of the mask.
     polygon : (N, 2) array_like
         The polygon coordinates of shape (N, 2) where N is
-        the number of points.
+        the number of points, and the column order is y, x.
```

Comment:
```suggestion
        the number of points. The coordinates are (row, column).
```

I would avoid speaking in terms of x and y, since we follow the matrix convention (where you count rows downwards), not the graphing one (where the y axis points upwards).

Reference:
https://scikit-image.org/skimage-tutorials/lectures/three_dimensional_image_processing.html#introduction-to-three-dimensional-image-processing